src/sagemaker/workflow/_utils.py

```diff
@@ -205,18 +205,19 @@ class _RegisterModelStep(Step):
             Estimator's training container image will be used (default: None).
         compile_model_family (str): Instance family for compiled model, if specified, a compiled
             model will be used (default: None).
+        container_def_list (list): A list of container defintiions.
         **kwargs: additional arguments to `create_model`.
     """
 
     def __init__(
         self,
         name: str,
         estimator: EstimatorBase,
-        model_data,
         content_types,
         response_types,
         inference_instances,
         transform_instances,
+        model_data=None,
```

Comment:
is this change necessary?

Reply:
> is this change necessary?

Yes, now that we are passing model list, this input may be empty, hence marked it as None.